EpiAware/src/EpiObsModels/modifiers/PrefixObservationModel.jl

```diff
@@ -0,0 +1,29 @@
+@doc raw"
+    Generate an observation model with a prefix. A lightweight wrapper around `EpiAwareUtils.prefix_submodel`.
+
+    # Constructors
+    - `PrefixObservationModel(model::M, prefix::P)`: Create a `PrefixObservationModel` with the observation model `model` and the prefix `prefix`.
+    - `PrefixObservationModel(; model::M, prefix::P)`: Create a `PrefixObservationModel` with the observation model `model` and the prefix `prefix`.
+
+    # Examples
+    ```julia
+    using EpiAware
+    observation_model = PrefixObservationModel(Poisson(), \"Test\")
+    obs = generate_observations(observation_model, 10)
+    rand(obs)
+    ```
+"
+@kwdef struct PrefixObservationModel{M <: AbstractTuringObservationModel, P <: String} <:
```

Comment:
Do we really need different classes and methods for latent and observation models? It seems pretty close to copy and paste? Is the hangup that `prefix_submodel` is taking more postional args?

Reply:
Resolved face to face